manifests/getambassador.io_v2_mapping_web-svc.yaml

```diff
@@ -8,3 +8,14 @@ spec:
   rewrite: /api/
   resolver: endpoint
   service: web-svc:8080
+---
+apiVersion: getambassador.io/v3alpha1
+kind: Mapping
+metadata:
+  name: web-svc-openapi
+  namespace: emojivoto
+spec:
+  service: web-svc:8080
+  prefix: /api/openapi
+  docs:
```

Comment:
I don't think you need this mapping. You should add doc on the web mapping : 

```yaml
apiVersion: getambassador.io/v2
kind: Mapping
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"getambassador.io/v2","kind":"Mapping","metadata":{"annotations":{},"name":"web","namespace":"emojivoto"},"spec":{"prefix":"/api/","resolver":"endpoint","rewrite":"/api/","service":"web-svc:8080"}}
  creationTimestamp: "2022-07-28T13:26:39Z"
  generation: 3
  name: web
  namespace: emojivoto
  resourceVersion: "585867"
  uid: 4445cbb3-81dd-400d-8312-b0377ee389ec
spec:
  docs:
    path: /openapi
  prefix: /api/
  resolver: endpoint
  rewrite: /api/
  service: web-svc:8080
```

Reply:
![image](https://user-images.githubusercontent.com/16439060/181518833-49783f21-f92d-4ca7-9761-a55b0cf18227.png)

Otherwise, for the leaderboard for example, the dev portal tries to request the following: 
```bash
curl -X 'GET' \
  'http://34.67.2.152/api/openapi/leaderboard' \
  -H 'accept: application/json'
```

Reply:
Also works with legacy dev portal, so this is the way :o)

![image](https://user-images.githubusercontent.com/16439060/181519437-0f07c060-104f-4294-9505-2a5fd480256a.png)